src/Symfony/Component/Intl/Countries.php

```diff
@@ -62,6 +100,18 @@ public static function getName(string $country, string $displayLocale = null): s
         return self::readEntry(['Names', $country], $displayLocale);
     }
 
+    /**
+     * Get country name from alpha3 code.
+     *
+     * @throws MissingResourceException if the country code does not exists
+     */
+    public static function getAlpha3Name(string $alpha3Code, string $displayLocale = null): string
+    {
+        $alpha2Code = self::getAlpha2Code($alpha3Code);
+
+        return self::readEntry(['Names', $alpha2Code], $displayLocale);
```

Comment:
`return self::getName(self::getAlpha2Code($alpha3Code))` without `@throws` here

Reply:
How can it be without `@throws` ?

Reply:
AFAIK we dont document indirect throws

Reply:
That is news to me. Thought a method had to document all throws, no matter where they come from.